template/chat.html

```diff
@@ -34,6 +34,9 @@
         }
       },
 
+      /*
+        writes a message in the chat box.
+      */
       sendMessage: function(user, message) {
         if (message != '')
```

Comment:
I'm old school. To me an empty string is a non null pointer to '\0'. Why not having to care about spaces if you want to specifically? It doesn't make sense to me. Anyway, we cannot change this behaviour so I guess we have to deal with it.